libs/components/ag-grid/src/lib/styles/_base.scss

```diff
@@ -0,0 +1,211 @@
+@use '~ag-grid-community/styles' as ag;
+@use 'libs/components/theme/src/lib/styles/variables' as variables;
+@use './variables' as sky-ag-grid-variables;
+
+@include ag.grid-styles(
+  (
+    themes: (
+      sky-default: (
+        background-color: variables.$sky-color-gray-01,
+        row-hover-color: darken(variables.$sky-background-color-selected, 10%),
+        border-color: variables.$sky-border-color-neutral-medium,
+        cell-horizontal-padding: 12px,
+        font-family: variables.$sky-font-family,
+        font-size: 15px,
+        foreground-color: variables.$sky-text-color-default,
+        header-background-color: variables.$sky-background-color-page-default,
+        header-height: 37px,
+        input-focus-border-color: variables.$sky-highlight-color-info,
+        row-border-color: variables.$sky-border-color-neutral-medium,
+        row-height: 38px,
+        odd-row-background-color: variables.$sky-background-color-page-default,
+        range-selection-border-color: variables.$sky-highlight-color-info,
+        secondary-foreground-color: variables.$sky-text-color-deemphasized,
+      ),
+      sky-modern-light: (
+        border-color: variables.$sky-theme-modern-border-color-neutral-medium,
+        cell-horizontal-padding: 12px,
+        font-family: variables.$sky-font-family,
+        font-size: 16px,
+        foreground-color: variables.$sky-text-color-default,
+        header-background-color:
+          variables.$sky-theme-modern-background-color-page-default,
+        header-height: 60px,
+        input-focus-border-color:
+          variables.$sky-theme-modern-border-color-primary-dark,
+        row-border-color:
+          variables.$sky-theme-modern-border-color-neutral-medium,
+        row-height: 60px,
+        range-selection-border-color:
+          variables.$sky-theme-modern-border-color-primary-dark,
+        secondary-foreground-color:
+          variables.$sky-theme-modern-font-deemphasized-color,
```

Comment:
☝️ Can any of these SCSS variables be replaced with our custom CSS properties?

Reply:
I tried this, but the SASS mixin they provide both validates that the value is a color and does color manipulations (e.g. `darken`) which won't work with custom properties.